<ui version="4.0" >
 <class>CutPanelBase</class>
 <widget class="QDialog" name="CutPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cut Management Panel</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1_2" >
         <property name="text" >
          <string>Current cut class:  </string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CutClassLabel" >
         <property name="text" >
          <string>none</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer14" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel1" >
         <property name="text" >
          <string>version</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cutverLabel" >
         <property name="text" >
          <string>none</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="TabWidg" >
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Main</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="Q3ListView" name="ClassList" >
               <column>
                <property name="text" >
                 <string>Cut Classes</string>
                </property>
                <property name="clickable" >
                 <bool>false</bool>
                </property>
                <property name="resizable" >
                 <bool>true</bool>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="IgnoreBadPix" >
               <property name="text" >
                <string>ignore defect pixels for module cuts</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="margin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="TextLabel2" >
                 <property name="text" >
                  <string>Selected:</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RenameButton" >
                 <property name="text" >
                  <string>Rename</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="RenameEdit" />
             </item>
             <item>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QPushButton" name="NewButton" >
                   <property name="text" >
                    <string>New</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="spacer9" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>21</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cutdefBox" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize" >
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QPushButton" name="DeleteButton" >
                   <property name="text" >
                    <string>Delete</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="spacer7" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>151</width>
                     <height>31</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QPushButton" name="LoadButton" >
                   <property name="text" >
                    <string>Load</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="Spacer4" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>168</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QPushButton" name="SaveButton" >
                   <property name="text" >
                    <string>Save</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="spacer8" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>151</width>
                     <height>21</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="Spacer5" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>239</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="SCButt" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Save+Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer11" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SaveButt" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer11_2_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ResetButt" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer11_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CloseButt" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
